### ISSUE TITLE
Create Sponsorship Prospectuses

### ISSUE THREAD

**Issue by EndangeredMassa** [ember-chicago/planning#13]
Create a Sponsorship Prospectus that offers various items for a price.

Possible items:
- recorded sessions: logo on screen
- explicit thanks to the audience from organizers at the start and end of the conferences
- signage
- badge logo
- website logo
- ember jobs board listings
- conference passes
- email ad
- conference tshirt logo

Examples:
- https://emberconf.com/become-a-sponsor.html
- https://tilde.wufoo.com/forms/emberconf-2018-a-la-carte-sponsorships/
- http://embercamp.com/files/embercamp-2017-prospectus.pdf

---

Blocked by: https://github.com/ember-chicago/planning/issues/8
- we need to figure out what JPMC wants before we can offer things to other sponsors

On complete: Create a new issue to reach out to each sponsor that has shown interest to see what they'd like to contribute.
